hack/osd-csi.yaml

```diff
@@ -209,6 +209,19 @@ spec:
       hostNetwork: true
       hostPID: false
       containers:
+        - name: osd
+          image: quay.io/openstorage/osd:latest
+          imagePullPolicy: Never
```

Comment:
We manually load the image in the shell script using Kind, instead of pushing to docker